index.js

```diff
@@ -3,61 +3,45 @@
 /* Expose. */
 module.exports = parse;
 
-/* Characters */
-var dot = '.'.charCodeAt(0);
-var hash = '#'.charCodeAt(0);
```

Comment:
Using `charCodeAt` is faster than `charCode` or bracket notation.
And this is code that could be run a lot, so it could matter a little.